.github/workflows/calens.yml

```diff
@@ -29,5 +29,5 @@ jobs:
       - name: Commit files
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
-          commit_message: Calens changelog updated
+          commit_message: "docs: Calens changelog updated"
```

Comment:
I guess you wrote `Calens` with capital letter because it is a proper noun, right?

Reply:
I just let the same message we had before, adding the prefix. Did not think on switching to non-capital, but i can change it

Reply:
Since we talked about writing everything in lowercase letters (except for proper nouns I guess), I let the decision in your hands. If you think it is just a normal message, then I'd write it in lowercase, but if you consider it as a proper noun, then it's ok that it has a capital letter

Reply:
as i understood, lowercase was intended for the pretty first letter in the message, in the prefix (the `d` of `docs`). I will move to lowercase anyway, but we should keep the same criteria in every message. Not a problem i guess.

Reply:
Yes, let's keep also the first letter of the description per se in lowercase as well, as agreed and as they show in the official [Examples](https://www.conventionalcommits.org/en/v1.0.0/#examples) 👍